packages/label/src/base-types.ts

```
@@ -0,0 +1,10 @@
+import type { ViewStyle } from 'react-native';
+
+type BaseRootProps = {
+  children: React.ReactNode;
+  style?: ViewStyle;
+};
+
+type BaseTextProps = {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)